pkg/config/config_test.go

```diff
@@ -276,6 +326,90 @@ func TestNewCreateChannelTxFailure(t *testing.T) {
+			err: errors.New("failed to create default config template: channel template config must contain " +
+				"an application section"),
+		},
+		{
+			testName: "When creating the default config template with no Admins policies defined fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				delete(profile.Policies, AdminsPolicyKey)
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: no Admins policy defined"),
+		},
+		{
+			testName: "When creating the default config template with no Readers policies defined fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				delete(profile.Policies, ReadersPolicyKey)
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: no Readers policy defined"),
+		},
+		{
+			testName: "When creating the default config template with no Writers policies defined fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				delete(profile.Policies, WritersPolicyKey)
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: no Writers policy defined"),
+		},
+		{
+			testName: "When creating the default config template with an invalid ImplicitMetaPolicy rule fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				profile.Policies[ReadersPolicyKey].Rule = "ALL"
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: invalid implicit meta policy rule: 'ALL' error: expected two space separated " +
+				"tokens, but got 1"),
+		},
+		{
+			testName: "When creating the default config template with an invalid ImplicitMetaPolicy rule fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				profile.Policies[ReadersPolicyKey].Rule = "ANYY Readers"
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: invalid implicit meta policy rule: 'ANYY Readers' error: unknown rule type " +
+				"'ANYY', expected ALL, ANY, or MAJORITY"),
+		},
+		{
+			testName: "When creating the default config template with SignatureTypePolicy and bad rule fails",
+			profileMod: func() *Profile {
+				profile := baseProfile()
+				profile.Policies[ReadersPolicyKey].Type = SignaturePolicyType
+				profile.Policies[ReadersPolicyKey].Rule = "ANYY Readers"
+				return profile
+			},
+			err: errors.New("failed to create default config template: failed to create new channel group: " +
+				"failed to add policies: invalid signature policy rule: 'ANYY Readers' error: Cannot transition " +
+				"token types from VARIABLE [ANYY] to VARIABLE [Readers]"),
```

Comment:
After looking at the production code, I wanted to see what this error looked like.  Sadly, it looks like I thought it would. The error from `FromString` is not particularly consumable.

Later in this cycle we are going to have to spend some time on errors. For now, this is fine.